assets/admin/style-overwrites/login.scss

```diff
@@ -0,0 +1,7 @@
+/* import original stylesheet to keep original styling */
+@import 'sulu-admin-bundle/containers/Login/login.scss';
+
+/* overwrite style for specific elements */
+.login-container {
+    background-color: lightpink;
+}
```

Comment:
If there is a need to overwrite the `colors.scss` have a look at: https://github.com/sulu/sulu/issues/6820#issuecomment-1258152906